geospaas_harvesting/crawlers.py

```diff
@@ -79,13 +79,13 @@ def handle_starttag(self, tag, attrs):
                     self._links.append(attr[1])
 
 
-class OpenDAPCrawler(Crawler):
-    """Crawler for OpenDAP resources"""
-
-    LOGGER = logging.getLogger(__name__ + '.OpenDAPCrawler')
-    FOLDERS_SUFFIXES = ('/contents.html')
-    FILES_SUFFIXES = ('.nc', '.nc.gz')
-    EXCLUDE = ('?')
+class BaseCrawler(Crawler):
+    """Base class (parent class) for all types of
+    crawlers which are used for various online data providers"""
```

Comment:
Be careful, this is not the parent class for all crawlers, just for repository which expose a directory-like structure like OpenDAP or Thredds. For example, repositories which expose a REST API don't fit in this category.
(And the `Crawler` class is the parent class for all crawlers)

```suggestion
class WebDirectoryCrawler(Crawler):
    """
    Parent class for crawlers used on repositories which expose a directory-like structure
    in the form of HTML pages
    """
```

Reply:
Done in the next commit